src/OrleansServiceBus/Providers/Streams/EventHub/EventHubSequenceToken.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+using System;
+using System.Globalization;
+using Orleans.Providers.Streams.Common;
+
+namespace Orleans.ServiceBus.Providers.Streams.EventHub
+{
+    /// <summary>
+    /// Event Hub messages consist of a batch of application layer events, so EventHub tokens contain three pieces of information.
+    /// Offset - this is a unique value per partition that is used to start reading from this message in the partition.
+    /// SequenceNumber - EventHub sequence numbers are unique ordered message IDs for messages within a partition.  
+    ///   The SequenceNumber is required for uniqueness and ordering of EventHub messages within a partition.
+    /// batch offset - Since each EventHub message may contain more than one application layer event, this value
+    ///   indicates which application layer event this token is for, within an EventHub message.  It is required for uniqueness
+    ///   and ordering of aplication layer events within an EventHub message.
+    /// </summary>
+    [Serializable]
+    internal class EventHubSequenceToken : EventSequenceToken
+    {
+        public string Offset { get; private set; }
```

Comment:
maybe call this EHOffset? Or HubOffset? So it's clear this is a part of EH impl, and not Orleans stream sequence?